src/pombot/index.js

```diff
@@ -11,16 +11,19 @@ import statusCommand from './commands/status';
 // create bot
 const bot = createSlackBot({
   name: 'Pombot',
-  slack: {
-    rtmClient: new RtmClient(config.tokens.pombot, {
-      dataStore: new MemoryDataStore(),
-      autoReconnect: true,
-    }),
-    webClient: new WebClient(config.tokens.pombot),
+  getSlack() {
```

Comment:
why `getSlack`, this seems strange inside `createSlackBot`. will there be a `getIRC` if you make an IRC bot?